app/src/main/res/layout/list_item_book.xml

```diff
@@ -32,14 +42,14 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:textSize="@dimen/primary_text_size"
-            src:text="Sample text" />
+            src:text="Sample text"/>
 
         <TextView
             android:id="@+id/tv_book_author"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:textSize="@dimen/secondary_text_size"
-            src:text="Sample text" />
```

Comment:
AAAaaa!! Tu jest tekst wpisany tak po prostu :( 
Nie ładnie... :D

Reply:
To też nie moje